### ISSUE TITLE
[BUG] Document created w/o id inside entity  does not contain it after fetching using fireorm

### ISSUE THREAD

**Issue by skalashnyk** [wovalle/fireorm#119]
He there!

We need some workaround for documents that created without `id` inside js object - see example.

```
import * as admin from 'firebase-admin'

@Collection()
class Band {
  id: string;
  name: string;
}

async function createBand(band: Band): string {
  const doc = await admin.firestore().collection('bands').add(band);
  return doc.id;
}

async function getById(id: string): Promise<Band> {
  const doc = await admin.firestore().collection('bands').doc(id);
  return {id: doc.id, ...doc.data()};
}

// Assume, that we used firestore native sdk to create doc:
const band = new Band();
band.name = 'Nirvana';
const bandId = await createBand(band);

// then we decided to use fireorm
const bandFromFirestore = await getRepository(Band).findById(bandId);

expect(bandFromFirestore).to.be.instanceof(Band); // true
expect(bandFromFirestore.id).not.to.be.undefined; // error!

```
I've created pull request (/w tests) that resolved this - #120

**Comment by zhirschtritt** [comment]
lgtm

**Comment by wovalle** [comment]
Closed in #124
